src/shared/bg_parse.cpp

```diff
@@ -1182,11 +1182,12 @@ void BG_ParseClassAttributeFile( const char *filename, classAttributes_t *ca )
 			ca->jumpMagnitude = atof( token );
 			defined |= JUMPMAGNITUDE;
 		}
-		else if ( !Q_stricmp( token, "cost" ) )
+		// "cost" is legacy. It has been replaced by "price" to increase consistency in interfaces.
+		else if ( !Q_stricmp( token, "cost" ) || !Q_stricmp( token, "price" ) )
```

Comment:
If you must use the same word everywhere, why not use the one which is already in config files, so that changes to packages are not needed?

Reply:
As said through the mail reply (which is not nicely integrated):

I would prefer to get rid completely of one of the terms, but that requires patching 2 repos, which is why I kept compat.
There are probably other things like that, I'm almost sure I remember things having same effect but different wording between aliens and humans weapons, with no real reason.
I would like to at least get rid of those things at some point, but the price/cost is one which striked me more I suppose.